src/WinRT.Runtime/Projections/IPropertyValue.net5.cs

```diff
@@ -1180,10 +1180,15 @@ private static unsafe int Do_Abi_get_Type_0(IntPtr thisPtr, global::Windows.Foun
             global::Windows.Foundation.PropertyType value;
             global::System.Type managedType = obj.GetType();
             bool isArray = managedType.IsArray;
+            bool isEnum = managedType.IsEnum;
```

Comment:
Can we do the same change here: IPropertyValue.netstandard2.0.cs

Reply:
Done.